### PR TITLE
Single isocorr

### LINKED ISSUE #431: Add example data from single isotope labeled experiments processed using IsoCorr
### Feature Inspiration

While most experiments processed using [IsoCorr13C15N](https://github.com/xxing9703/isocorr13C15N) use tracer compounds with multiple isotope labels (both C and N), IsoCorr can process singly labeled data. That type of data should be supported (#349) and thus having an example of that data will be useful.

### Feature Description

A study with data corrected by "isocorr_CN" is available in the [Google Drive folder `AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected`](https://drive.google.com/drive/folders/1zyR6zaWikpooX2YKhCYtlBCgz9lwqzJG?usp=sharing). This should be added in "study" format to the `example_data` folder.

### Comment

-----

### Assumptions

- List of assumptions made WRT the code
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- List of conditions to be met for the feature
- E.g. Every column/row must display a value, i.e. cannot be empty

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected/Changed Components

- Files
- Environment variables
- External executables
- Database tables
- Cron job settings
- Etc.

#### GUI Change description

Describe changes the user will see.

#### Code Change Description *(Pseudocode optional)*

Describe code changes planned for the feature.

#### Tests

A test should be planned for each requirement (above), where possible.

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

Added a singly-labeled isocorr run as a "study" to the `example_data` directory.  Also added an option to indicate the format.  Edited line endings for superlinter.

## Affected Issue Numbers

- Resolves #431

## Code Review Notes

I basically just copied what appears to already have been put into "study" format, [from the google drive](https://drive.google.com/drive/folders/1zyR6zaWikpooX2YKhCYtlBCgz9lwqzJG?usp=sharing).  I'm not sure of the correctness of the "studification" of the files, since it can't be loaded yet (vis-a-vis #349).  #349 will run a test by loading this data.

Implementing #349 depends on this PR (issue #431) and #431 depends on the option `isocorr_format` to be implemented in #349, i.e. the issues are interdependent, so this PR adds the format option to the yaml file even though it does not yet exist.

## Checklist

- [x] All issue requirements satisfied (or no linked issues)
- [x] [Linting passes](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting).
- [x] [Migrations created & committed *(or no model changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)
- [x] [Tests implemented *(or no code changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
- [ ] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)

### CODE REVIEW

**Review by jcmatese — APPROVED** [review]
Fine by me.  Not a big fan of spaces in the filenames, but might be a hassle to modify, if that is what the user provided.  Can always modify as we go...

**Comment by hepcat72** [comment]
> Fine by me. Not a big fan of spaces in the filenames, but might be a hassle to modify, if that is what the user provided. Can always modify as we go...

Yeah, I debated changing them, but elected to see if anyone would say whether or not it should be changed in the review, because it looked like the directory had already been put in "study" format.

**Review by fkang-pu — APPROVED** [review]
It's a minor issue, but I also don't like to use space in file names.

File: DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml [hunk 1]
```diff
@@ -13,3 +13,4 @@ accucor_data:
   date: "2021-06-06"
   researcher: "Xianfeng Zeng"
   new_researcher: false
+  isocorr_format: true
```
Comment (lparsons) [inline DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: I don't think that the format should be a study wide parameter, but instead something set "per data file". In other words, in the `DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/loading.yaml` file.

**Review by lparsons — APPROVED** [review]
Overall, this looks fine to merge, though I suspect a few things will need to get updated. In addition to the inline comments, there are a few samples with missing tissue (should be `feces` I believe, which I think is in the list of tissues already). Minor issues, so update those and then merge, or fix later.

File: DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/loading.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  accucor_files:
+    - name: "./13C-val tissues metabolites results - No D-Val_cor.csv"
+    - name: "./13C-val tissues metabolites results 146,148,149 no D-val_cor.csv"
+    - name: "./Serum results_cor.csv"
+  msrun_protocol: "Hilic25min_NegativeMode"
+  date: "2021-12-07"
+  researcher: "Asael Roichman"
+  new_researcher: false
+  skip_samples:
+    - "blank"
```
[the remark this answers is not in the document]
Comment (hepcat72) [reply DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/loading.yaml:15]: Yeah, I figured having this merged will help with the loading code.

File: DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml (hunk 1 above)
Comment (hepcat72) [reply DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: Yeah, that's what I thought, but reading into your comment (incorrectly), I figured I was mistaken. Thanks for the quick reply.

Comment (hepcat72) [reply DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: What's a good documentation source that would have revealed the fact that processing the yaml would result in multiple runs of the command line loading tool (and explain how it operates)? Is this a feature of argparse, django, something else, or is it custom code? I'm curious.

Comment (lparsons) [reply DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: I presume you are asking about how the `load_study` command works. I don't think there is extensive documentation, but it's pretty simple. It just iterates over a YAML file and calls the individual commands as needed.
* [`load_study.py`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/DataRepo/management/commands/load_study.py) - The code is relatively straightforward
* [`load_study.yaml`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/DataRepo/schemas/load_study.yaml) - Schema definition for the YAML file.
* Various examples of available in the [`tracebase-rabinowitz-data` repository](https://github.com/PrincetonUniversity/tracebase-rabinowitz-data). One example that uses a different paramter for each file is [serine-glycine-free-glucose-infusion](https://github.com/PrincetonUniversity/tracebase-rabinowitz-data/blob/main/serine-glycine-free-glucose-infusion/loading.yaml) which requires a different prefix for each data file.

Comment (hepcat72) [reply DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: Ah, OK.  I was in the `load_accucor_msruns.py` headspace. It's essentially a Monday, today, after all. lol

Comment (hepcat72) [reply DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml:16]: I just realized that this value is in the wrong file (`obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml`).  It was intended to be in the file associated with the isocor csv files.  I was wondering why I didn't see it in the load code branch and figured I must have branched before the last commit. After merging, I was just looking over the file browser on the left and saw this unexpected file... *Now* I know what you meant by your "study wide parameter" comment. I thought you were referring to the location of the parameter in the yaml structure of the study.

- [ ] Remove isocorr_format from the obob study yaml.

File: DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12 changelog
```
Comment (hepcat72) [inline DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/CHANGES.md:1]: @lparsons - BTW, do these modified files need to be updated on the google drive, added to 
tracebase-rabinowitz-data, or both?

Comment (lparsons) [reply DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/CHANGES.md:1]: Good question. The datasets that need to get added each have RT tickets open.

**Pull request merged**